coreconfig/config.go

```diff
@@ -48,9 +48,9 @@ var (
 )
 
 const (
-	autopilotTimer         string = "snappy-autopilot.timer"
-	autopilotTimerEnabled  string = "enabled"
-	autopilotTimerDisabled string = "disabled"
+	autoUpdateTimer         string = "snappy-autopilot.timer"
```

Comment:
Should we disable the file as well? i.e. snappy-autopilot.timer -> snappy-autoupdate.timer ?